patches/minecraft/net/minecraft/client/resources/ResourcePackRepository.java.patch

```diff
@@ -0,0 +1,22 @@
+--- ../src-base/minecraft/net/minecraft/client/resources/ResourcePackRepository.java
++++ ../src-work/minecraft/net/minecraft/client/resources/ResourcePackRepository.java
+@@ -310,9 +310,10 @@
+ 
+     private boolean func_190113_a(String p_190113_1_, File p_190113_2_)
+     {
++        InputStream is = null;
+         try
+         {
+-            String s = DigestUtils.sha1Hex((InputStream)(new FileInputStream(p_190113_2_)));
```

Comment:
You use try-with in all the code we control where size doesn't matter.. but you don't feel like doing it in the patches where size does?